dev-libs/libnl/files/libnl-3.11-add-xfrm-iface-id.patch

```diff
@@ -0,0 +1,265 @@
+diff --git a/include/netlink/xfrm/sa.h b/include/netlink/xfrm/sa.h
```

Comment:
Patches should always have a link to the commit they came from upstream and any relevant context. I recommend fetching them with the .patch suffix on the commit link so you can paste the full header.

But why are we backporting an _API addition_? This needs justification. We don't pull in all patches after a tag. Just ones that look important.